data/data/bootstrap/baremetal/files/etc/containers/systemd/metal3-baremetal-operator.container.template

```diff
@@ -0,0 +1,35 @@
+[Unit]
+Description=Metal3 deployment service
+BindsTo=ironic-volume.service
+Requires=build-metal3-env.service wait-iptables-init.service image-customization.service build-ironic-env.service
+Wants=network-online.target ironic-httpd.service ironic-inspector.service ironic-ramdisk-logs.service
+After=network-online.target ironic-volume.service build-ironic-env.service wait-iptables-init.service provisioning-interface.service ironic-dnsmasq.service image-customization.service
```

Comment:
missing After=build-metal3-env.service

Reply:
I think we only need network-online, build-metal3-env and maybe image-customization here

Reply:
Done